Server/src/Server.Domain/Entities/Book.cs

```diff
@@ -71,12 +71,12 @@ public void SubtractReviewFromRating(double reviewRating)
         AverageRating = Math.Round((averageRating * reviewCount - reviewRating) / (reviewCount - 1), 2);
     }
     
-    public void UpdateReviewRating(double oldReviewRating, double newReviewRating)
+    public void UpdateReviewRating(ICollection<Review> reviews, double oldReviewRating, double newReviewRating)
     {
-        var reviewCount = Reviews.Count;
+        var reviewCount = reviews.Count;
         var averageRating = AverageRating ?? 0;
         
-        if (reviewCount <= 1)
+        if (reviewCount < 1)
         {
             AverageRating = null;
             return;
```

Comment:
Hmm, a jak nie ma żadnej opinii dotychczas i dodajemy nową to nie zostanie przypadkiem AvarageRating nullem?

Reply:
ale to jest Update, wiec zawsze istnieje przynajmniej jedna opinia. Ale dzieki za ten comment bo ten if statement i tak jest do wywalenia bo nigdy count nie bedzie mniejszy niz 1